packages/sources/fluent-finance/test/e2e/README.md

```diff
@@ -0,0 +1 @@
+Tests that run the adapter and send an end to end request using live API endpoints.
```

Comment:
This was probably autogenerated but if we don't have e2e tests should we remove this directory. Also makes sense to keep as a placeholder but thought I'd ask

Reply:
I kept as a placeholder. I can add one just so that there is something there.